docs/docs/addons-test-utils.md

```diff
@@ -46,19 +53,60 @@ var ReactTestUtils = require('react-addons-test-utils') // ES5
  - [`scryRenderedComponentsWithType()`](#scryrenderedcomponentswithtype)
  - [`findRenderedComponentWithType()`](#findrenderedcomponentwithtype)
 
-### Shallow Rendering (Experimental)
+* * *
 
-Shallow rendering is an experimental feature that lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+## Reference
 
-  - [`createRenderer()`](#createrenderer)
-  - [`render()`](#render)
-  - [`getRenderOutput()`](#getrenderoutput)
+### `createRenderer()`
```

Comment:
With this change, they are mixed together with all the other top-level APIs. For example, it seems like `getRenderOutput()` is a top-level export rather than a method on the object returned by `createRenderer()`.

I think we should group shallow rendering APIs separately in the reference, and make it clear from the titles that `render()` and `getRenderOutput()` are parts of `createRenderer()` rather than independent methods.